_i18n/en/resources/user-guides/weblate/cli_wallet_daemon_isolation_qubes_whonix.pot

```diff
@@ -0,0 +1,257 @@
+# Translation file for Weblate
+# Copyright (C) 2021,The Monero Project
+# This file is distributed under the same license as the monero-site repository.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2021-03-22 11:27+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:2
+msgid ""
+"{% include disclaimer.html translated=\"no\" translationOutdated=\"no\" %}"
+msgstr ""
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:4
+msgid ""
+"With [Qubes](https://qubes-os.org) + [Whonix](https://whonix.org) you can "
+"have a Monero wallet that is without networking and running on a virtually "
+"isolated system from the Monero daemon which has all of its traffic forced "
+"over [Tor](https://torproject.org)."
+msgstr ""
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:6
+msgid ""
+"Qubes gives the flexibility to easily create separate VMs for different "
+"purposes. First you will create a Whonix workstation for the wallet with no "
+"networking. Next, another Whonix workstation for the @daemon which will use "
+"your Whonix gateway as it's NetVM. For communication between the wallet and "
+"daemon you can make use of Qubes [qrexec](https://www.qubes-os.org/doc/"
+"qrexec3/)."
+msgstr ""
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md:8
+msgid "new paragraph of text that would mess up the entire format."
```

Comment:
Is this supposed to be here?

Reply:
No. It's a leftover from my tests. Nice catch :)